update

```diff
@@ -152,6 +152,22 @@ const serialize = async (data, file, factor = 1) => {
   return [json.length, compressed.length];
 };
 
+const createSetKey = (set) => {
+  const baseKey = [
+    formatString(set.level.toString()),
```

Comment:
clearly an over zealous use of formatString, you already know set is a number. also, you should just to `toID`-ing things instead of inventing a new identifier function

Reply:
Thanks, didn't know about Dex.toID.